Cargo.toml

```diff
@@ -55,7 +55,7 @@ repository = "https://github.com/kanidm/kanidm/"
 
 # ldap3_client = { path = "../ldap3/client" }
 # ldap3_proto = { path = "../ldap3/proto" }
-# ldap3_client = { git = "https://github.com/kanidm/ldap3.git" }
+ldap3_client = { git = "https://github.com/fooker/ldap3.git", branch = "pr/get_ava" }
```

Comment:
This probably needs to be removed?

Reply:
I think so. But this needs to have a release of ldap3 first, right?

Reply:
Ah, didn't see you'd PR'd over there because it wasn't mentioned. We'll need to block this PR until ldap3 gets a release, in that case.

Reply:
If you can update the dep to 0.4.4 you should be good to go.

Reply:
Done.